sphinx/index.rst

```diff
@@ -64,3 +64,10 @@ Indices
 
 * :ref:`genindex`
 * :ref:`modindex`
+* :ref:`terms`
```

Comment:
Introduces new build `WARNING: toctree contains reference to nonexisting document u'terms'`?